### PR TITLE
fix for one sample + test

### PR DESCRIPTION
**Pull request by apmellot**
It should work for one sample now @agramfort @dengemann

### CODE REVIEW

File: meegpowreg/tests/test_pipelines.py [hunk 1]
```diff
@@ -58,3 +60,22 @@ def test_pipelines(toy_data):
         categorical_interaction="drug")
     y_bin = np.sign(y - np.mean(y))
     regressor.fit(X_df, y_bin)
+
+
+def test_one_sample():
```
Comment (agramfort) [inline meegpowreg/tests/test_pipelines.py:65]: can you move this test in test_spatial_filters.py ?

**Review by agramfort — COMMENTED** [review]
just a nitpick @apmellot 

did you check if it fixes our pb on the real data?

**Comment by apmellot** [comment]
@agramfort Yes I checked and it works with real data :heavy_check_mark:

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx a ton @apmellot !

**Comment by dengemann** [comment]
Thanks the two of you!!
________________________________

**Pull request merged**